app/lib/frontend/templates/views/publisher/admin_page.mustache

```diff
@@ -15,9 +15,17 @@
 <div id="-admin-authorized">
   <h2>Admin publisher: {{publisher_id}}</h2>
   <h4>Description</h4>
-  <p>
+  <div>
     <textarea id="-publisher-description" class="pub-input" rows="16">{{description}}</textarea>
-  </p>
+  </div>
+  <h4>Website URL</h4>
+  <div>
+    <input id="-publisher-website-url" class="pub-input" value="{{website_url}}" />
+  </div>
+  <h4>Contact e-mail</h4>
+  <div>
+    <input id="-publisher-contact-email" class="pub-input" value="{{contact_email}}" />
```

Comment:
Shouldn't we be using `<form>` and `<label for='-publisher-contact-email'>Contact e-mail<label/>`, instead of a `<h4>`.

Reply:
Maybe, but form also implies ability to send it as a separate request, and we are not doing that, as we need to augment the request with the auth header, so it won't work without JS anyway.